src/Uno.UI.RuntimeTests/Tests/HotReload/Frame/HRApp/Tests/Given_Frame_DataContext.cs

```diff
@@ -78,6 +78,7 @@ public async Task Check_Can_Change_Page1_With_DataContext()
 	/// Navigate back to Page1
 	/// </summary>
 	[TestMethod]
+	[Ignore("This test is no longer valid with WinUI Frame #17739")]
```

Comment:
Could this be causing HR regressions that are visible to the user?

Reply:
I have created an issue to adjust the tests, but it technically should not, as the test assumes the same instance, but as long as HR updates the type itself, the new instance should have the changes preserved correctly